test/y2partitioner/widgets/encrypt_method_options_test.rb

```diff
@@ -46,10 +48,12 @@
   describe "#refresh" do
     let(:fake_random_swap_options) { CWM::Empty.new("__fake_plain_options__") }
     let(:fake_luks1_options) { CWM::Empty.new("__fake_luks1_options__") }
+    let(:fake_pervasive_options) { CWM::Empty.new("__fake_pervasive_options__") }
 
     before do
       allow(Y2Partitioner::Widgets::SwapOptions).to receive(:new).and_return(fake_random_swap_options)
       allow(Y2Partitioner::Widgets::LuksOptions).to receive(:new).and_return(fake_luks1_options)
+      allow(Y2Partitioner::Widgets::PervasiveOptions).to receive(:new).and_return(fake_pervasive_options)
     end
 
     it "generates the new content based on the selected method" do
```

Comment:
np: I would probably write separate tests for each case. But they are simple enough, so let's keep them as they are.